src/utilities/effects.ts

```diff
@@ -1253,7 +1255,7 @@ const effects = {
 
       const { confirm } = await showConfirmModal(
         'Delete',
-        'Are you sure you want to delete this dictionary?',
+        `Are you sure you want to delete the dictionary with ID: ${id}?`,
```

Comment:
small nitpick, here you don't have quotes around the `${id}` and below you have another `"${sequence.seq_id}"` with quotes

Reply:
Ah yeah, I can add quotes there, think it was a slightly different case because of the number but think it's best to be consistent.